elfeed-org.el

```diff
@@ -36,6 +36,8 @@
 (require 'org)
 (require 'dash)
 (require 's)
+(require 'cl-lib)
+(require 'xml)
```

Comment:
Updated now, but I found `elfeed` even didn't require `cl-lib` through full of `cl-loop` in it's code, so maybe as a bultin library, requre or not should be all OK~